libtelemetry/src/main/java/com/mapbox/android/telemetry/TelemetryService.java

```diff
@@ -273,9 +273,12 @@ private boolean locationPermissionCheck() {
     if (Build.VERSION.SDK_INT >= API_LEVEL_23) {
       return PermissionsManager.areLocationPermissionsGranted(this);
     } else {
-      int finePermission = PermissionChecker.checkSelfPermission(MapboxTelemetry.applicationContext,
-              Manifest.permission.ACCESS_FINE_LOCATION);
-      return checkFinePermission(finePermission);
+      if (MapboxTelemetry.applicationContext != null) {
```

Comment:
```suggestion
      if (MapboxTelemetry.applicationContext == null) {
         // Log error
         return false;
      }
```
Should we log error, such that we have a better understanding why permissions weren't granted?